build-system/server/new-server/transforms/modules/modules-transform.ts

```diff
@@ -18,18 +18,33 @@ import {PostHTML} from 'posthtml';
 import {URL} from 'url';
 import {isValidScript, ScriptNode} from '../utilities/script';
 import {CDNURLToLocalDistURL} from '../utilities/cdn';
+import {OptionSet} from '../utilities/option-set';
+import minimist from 'minimist';
+const argv = minimist(process.argv.slice(2));
 
 /**
  * Append a Module Script for a ScriptNode.
  * @param head
  * @param script
```

Comment:
update comment... although in saying that lets just get rid of these since its typescript anyways and not closure

Reply:
I got rid of the comment and just added params to the header.